src/mz/Readme.md

```diff
@@ -58,46 +59,60 @@ The configuration file stores all the available profiles. You can add your own a
 ```TOML
 ["profiles.PROFILE_NAME"]
 email = "your@email.com"
-secret = "YOUR_SECRET"
-client_id = "YOUR_CLIENT_ID"
+app_password = "YOUR_APP_PASSWORD"
 ```
 
-##### Paths
-Linux: `.config/mz/profiles.toml`
-macOS (Monterrey): `.config/mz/profiles.toml`
-Windows: `%UserProfile%\.config\mz\profiles.toml`
+#### Paths
+* Linux: `.config/mz/profiles.toml`
+
+* macOS (Monterrey): `.config/mz/profiles.toml`
+
+* Windows: `%UserProfile%\.config\mz\profiles.toml`
 
 Example:
 ```TOML
 ["profiles.production"]
 email = "account@example.com"
-secret = "e1620d58-b3f5-454c-bdc8-6ac07da79fd1"
-client_id = "f599e025-11b3-4fd8-876f-293787dca3c9"
+app-password = "mzp_e1620d58b3f5454cbdc86ac07da79fd1f599e02511b34fd8876f293787dca3c9"
```

Comment:
I assume this isn't a real one, right?

Reply:
In either case, I'd like to make it clear that under no circumstances will that password work (but the format is similar):

```suggestion
app-password = "mzp_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
```

Reply:
Correct assumption!